hack/crdgen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This script only generates minimal crd right ? If yes we name the script more specifically

Reply:
Renamed!